azure-pipelines.yml

```diff
@@ -166,12 +186,20 @@ jobs:
         plat: manylinux2010_x86_64
         python_ver: '38'
         python.version: '3.8'
+  variables:
+    CCACHE_DIR: $(Pipeline.Workspace)/ccache
 
   steps:
   - task: UsePythonVersion@0
     inputs:
       versionSpec: '$(python.version)'
 
+  - task: Cache@2
+    inputs:
+      key: 'ccache-wheels | $(Agent.OS) | "$(python.version)"'
```

Comment:
This has a different first key than the others, as caching boost compilation needs a lot of space and there is no need to mounting that cache in other builds.